lucille-core/src/main/java/com/kmwllc/lucille/stage/ExtractEntitiesFST.java

```diff
@@ -40,6 +44,10 @@
  *   all overlapping matches that start at that position. Defaults to false.</li>
  *   <li>stop_on_hit (Boolean, Optional) : Denotes whether this matcher should stop after one hit. Defaults to false.</li>
  *   <li>ignore_case (Boolean, Optional) : Denotes whether this Stage will ignore case determining when making matches. Defaults to false.</li>
+ *   <li>use_whitespace_tokenizer (Boolean, Optional) : If true, tokenization uses a whitespace tokenizer. If token_break_regex is provided
+ *   a pattern replace filter applies first to replace matches with a single space. Defaults to false.</li>
```

Comment:
should mention that false (or omitted) means the Lucene StandardTokenizer will be used

```
If true, tokenization uses Lucene's WhitespaceTokenizer. If token_break_regex is provided,
 a pattern replace filter applies first to replace matches with a single space. 
If false, tokenization uses Lucene's StandardTokenizer. Defaults to false.
```